starter-code/public/scripts/models/repo.js

```diff
@@ -6,8 +6,14 @@
   repos.all = [];
 
   repos.requestRepos = function(callback) {
-    // TODO: How would you like to fetch your repos? Don't forget to call the callback.
-
+    // Done: How would you like to fetch your repos? Don't forget to call the callback.
+    $.ajax({
+      url: 'https://api.github.com/users/brials/repos',
+      method: 'GET',
+      headers: {
+        Authorization: githubToken //eslint-disable-line
```

Comment:
so close! Authorization: 'token ' + githubToken should get you where you need to be